package.json

```
@@ -33,21 +33,22 @@
   },
   "homepage": "https://github.com/CrowdStrike/eslint-config-crowdstrike-node#readme",
   "dependencies": {
-    "eslint-config-crowdstrike": "^11.0.0",
     "eslint-plugin-json-files": "^4.3.0",
     "eslint-plugin-n": "^17.0.0",
     "globals": "^15.8.0"
   },
   "devDependencies": {
     "@crowdstrike/commitlint": "^8.0.0",
     "eslint": "^9.0.0",
+    "eslint-config-crowdstrike": "^11.0.0",
     "eslint-config-crowdstrike-node": "link:",
     "remark-cli": "^12.0.0",
     "remark-preset-lint-crowdstrike": "^4.0.0",
     "renovate-config-standard": "^2.0.0",
     "standard-version": "^9.0.0"
   },
   "peerDependencies": {
+    "eslint-config-crowdstrike": ">=1",
```

```suggestion
    "eslint-config-crowdstrike": ">=11",
```

I think we need the eslint 9 stuff at least.